Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -84,8 +92,14 @@ void BinaryBHLevel::computeTaggingCriterion(FArrayBox &tagging_criterion,
 {
     if (m_p.track_punctures)
     {
-        const vector<double> puncture_masses = {m_p.bh1_params.mass,
-                                                m_p.bh2_params.mass};
+        std::vector<double> puncture_masses;
+#ifdef USE_TWOPUNCTURES
```

Comment:
I don't think you need this exception for TP as in the SimulationParameters you set bh1_params.mass to these bare masses anyway, which I think is the right thing to do.

Reply:
The issue is that I wanted the TwoPunctures version to use the bare mass here (for consistency) independent of whether the bare or target masses were specified. In the latter case, `bh1_params.mass` and `bh2_params.mass` are set to the target masses (as these are all that is known at parameter loading time). Of course in practice the difference is small, but I'd prefer consistency.

Reply:
I see, ok then.